docusaurus/docs/cloud/advanced/database.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Database
+displayed_sidebar: cloudSidebar
+description: Configure your own database on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/database.html
+---
+
+# Database
+
+Strapi Cloud provides a pre-configured PostgreSQL database by default. However, you can also configure it to utilize an external SQL database, if needed.
+
+:::caution
+While supported, it is not recommended to use an external database with Strapi Cloud, unless there is an explicit need to do so. This is because Strapi Cloud provides a managed database that is optimized for Strapi. Using an external database may result in unexpected behavior and/or performance issues (e.g., network latency may impact performance). Strapi cannot provide security or support for external databases.
+
+:::
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.1+`.
+- Credentials for external database.
+- If using an existing database, the schema must match the Strapi project schema.
+
+:::
+
+## Configuration
+
+The project `./config/database.js` or `./config/database.ts` file must match the configuration found in the [environment variables in database configurations](https://docs.strapi.io/dev-docs/configurations/database#environment-variables-in-database-configurations) section.
+
+Before pushing changes, add environment variables to the Strapi Cloud project:
+
+1.  Log into Strapi Cloud and click on the corresponding project on the Projects page.
+2.  Click on the **Settings** tab and choose **Variables** in the left menu.
+3.  Add the following environment variables:
+
+    | Variable                           | Value            |
+    | ---------------------------------- | ---------------- |
+    | `DATABASE_CLIENT`                  | your_db          |
+    | `DATABASE_HOST`                    | your_db_host     |
+    | `DATABASE_PORT`                    | your_db_port     |
+    | `DATABASE_NAME`                    | your_db_name     |
+    | `DATABASE_USERNAME`                | your_db_username |
+    | `DATABASE_PASSWORD`                | your_db_password |
+    | `DATABASE_SSL`                     | true             |
```

Comment:
Should remove this line, if you specify the nested option this is inherently true